js/app.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var projectArray = [];
+function ProjectData(name, date, sitePath) {
+  this.name = name;
+  this.date = date;
+  this.sitePath = sitePath;
+  //this.push(projectArray);
+}
+var aboutMe = new ProjectData('About Me', 'Code Fellows 201 Week 1', 'url');
```

Comment:
Here you should be getting your information from SourceData.js and instantiate the constructor to store into a new array.